bindings/python/setup.py

```diff
@@ -193,8 +197,13 @@ def build_extension_with_b2(self):
             self.announce("project-config.jam contents:")
             self.announce(config.read())
             config.close()
+            args.append(f"python={sysconfig.get_python_version()}")
```

Comment:
As I understand it, this gets overridden by the way i've instantiated the python tool in the project-config, with the `libtorrent-python` feature

Reply:
if I have some other python version configured in `user-config.jam`, that other version may be the default and used. This specifies explicitly which version to use. Without this, it would not work on ubuntu for me (since I *do* have a `user-config.jam` with other python versions configured)

Reply:
I don't actually fully understand the purpose of `libtorrent-python` though. I thought it was there to force a build, or something

Reply:
The intent is to force the use of *our* python config from `project-config.jam`, disregarding `user-config.jam` or any other configs.

My understanding is `python.jam` will prefer configurations that match the "conditions" properties. So we create the fake feature `libtorrent-python` set to always-on, and hopefully any python configuration with `<libtorrent-python>on` gets preferred.

Or at least I assume that was the intent; I just migrated this from `bindings/python/Jamfile` to `project-config.jam`.

Apparently it doesn't work, if your `user-config.jam` overrode it. I don't know if there's something else I should be doing. I mostly don't understand the design of b2 and am confused by its documentation, but the parts I do understand I've found disappointing

Reply:
I see. the `libtorrent-python=on` command line argument is missing. It works once I add it. However, we still need to specify the correct python version. The version is an independent feature, and if the default is different than what we're building, this `libtorrent-python` feature won't be considered. It's only used as a "tie-breaker" if there's another python configuration with the same version.